tests/plugins/justify/justify.js

```diff
@@ -252,5 +252,87 @@ bender.test(
 		bot.setHtmlWithSelection( '[<p><span contenteditable="false">foo</span></p>]' );
 		bot.execCommand( 'justifyright' );
 		assert.areSame( '<p style="text-align:right;"><span contenteditable="false">foo</span></p>', bot.getData( true ) );
+	},
+
+	'test alignment on disabled elements paragraph': function() {
```

Comment:
Hmm… Looking through original issue, I've discovered that I completely forgot about `disallowContent` rule. Proposed fix should be working with it without any issues, but we could add unit test for it – just in case.